src/github_utils.py

```diff
@@ -19,48 +19,56 @@ def get_pull_request_diff():
     # Get diff content
     diff_url = pull_request.diff_url
     response = requests.get(diff_url)
-    print(response.text)
+    #print(response.text)
```

Comment:
The commented-out print statement should be removed.  Leaving commented-out code can clutter the codebase.

```suggestion
Remove the line: `#print(response.text)`
```